_drafts/2023-02-13-cheating-the-system-for-fun-and-profit.md

```diff
@@ -0,0 +1,141 @@
+test your (iOS) app without having to take out your iOS device... mostly. But
+we'll get to that.
+
+<!--more-->
+
+## Hover Text
+
+Hover Text is a macOS feature that let's you view text at larger sizes,
+typically using a modifier key and hovering over the text or element (hence the
+name). What's neat about it, is that is will show you the accessibility label
+of any element, not just text, including in the iOS simulator. Meaning Hover
+Text can be a rather quick and frictionless way to get an idea of elements not
+having an accessibility label, or having an awkward one. You can find Hover Text
+under System Settings (or System Preferences if you're not yet running macOS 13
+Ventura) > Accessibility > Zoom > Hover Text.
+
+![Hover Text in the macOS system settings.](./assets/blog-assets/(hover-text.png)
+
+And this is what Hover Text looks like in practice, here as seen in the
+simulator.
+
+![Hover Text in the simulator.](./assets/blog-assets/(hover-text-in-simulator.png)
+
+A neat start, but let's take a look at macOS VoiceOver next...
+
+## macOS VoiceOver?! I've never used that!
+
+So yeah, this trick is certainly a bit for power users. If you build apps for
+iOS, there's a chance you're not familiar with VoiceOver on macOS — I wasn't
+until joining the macOS Accessibility team at Apple. There's certainly a bit of
+a learning curve, which people familiar with iOS VoiceOver will probably know.
+
+System Settings > Accessibility > VoiceOver > Open VoiceOver Training... is a
+good place to start, but let's go over an even quicker quickstart.
+
+### macOS VoiceOver quickstart
+
+To turn on VoiceOver on macOS, given a keyboard with Touch ID, hold `⌘`,
+then triple-press the Touch ID button. You can also ask Siri to turn it on or
+off.
+
+Then, as you can imagine, VoiceOver navigation is going to be quite different
+from iOS. No touch screen, but a keyboard to interact with things. Oh, the
+possibilities!
+
+The first thing to know is there's the so called "VoiceOver modifier" keys. By
+default that'll be either Caps Lock or `⌃`+`⌥`. I've gotten used to the latter.
+To navigate, hold your modifier key(s), and use the arrow keys. That's like a
+right (or left) swipe on iOS, and honestly, you're starting to get off to the
+races. Try it out!
+
+---
+<br />
+
+Activating things like buttons, as you might have guessed, is done with the
+modifiers keys and the space bar.
+
+That concludes our quickstart for now, let's get back to the (fun and) profit.
+
+## (Fun and) profit
+
+Macs with Apple's M-chips can run iOS apps on the Mac — which is practically
+the same thing the simulator does, and how Catalyst apps work. And so we can
+leverage this with the Simulator app, simply by navigating our macOS VoiceOver
+cursor... to our iOS app!
+
+![Navigating into the iOS app running on the simulator using macOS VoiceOver.](./assets/blog-assets/(into-ios.mov)
+
+You can already tell the... awkward bits that come with this. No idea why the
```

Comment:
Perhaps worth using this as an example of why its really important to test on real devices before releasing?